hackathon_site/event/jinja2/event/application.html

```diff
@@ -0,0 +1,110 @@
+{% extends "event/form_base.html" %}
+
+{% block form_title %}Application{% endblock %}
+{% block form_button %}Submit{% endblock %}
+               
+{% block form_inputs %}
+    <h4 class="formH2 col s12">Personal</h4>
+    <div class="input-field col m7 s12">
+        <input id="birthday" type="text" class="validate">
+        <label for="birthday">birthday</label>
```

Comment:
Lowercase labels still spook me

Reply:
lol these dont matter cause it'll be Django inputa later right